addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -20,7 +20,11 @@
     resize: none;
     border-radius: $o-mail-rounded-rectangle-border-radius-lg;
     border: none;
-    overflow: auto;
+    overflow: hidden;
```

Comment:
This will remove the vertical scrollbar when typing a long message, which is not good.

Maybe ok if limited to overflow-x

Reply:
Works the same with `overflow-x` but it allow the scroll bar when needed, so it's good solution